packages/client/hmi-client/src/page/Home.vue

```diff
@@ -393,7 +396,7 @@ header > section > button {
 }
 
 .filter-and-sort label {
-	padding-right: 0.25rem;
+	padding-right: 0.25]rem;
```

Comment:
typo? `0.25]rem`?

Reply:
Yup, missed that one @mwdchang  - I'll fix.